### DISCUSSION TITLE
Reporting a bug? Open an issue, not a discussion.

### DISCUSSION THREAD

**Discussion by esantorella** [meta-pytorch/botorch#2423] [General]
BoTorch issues are a good fit for bug reports or feature requests. Generally, if your problem will require a code change to resolve, an issue may be a better fit.

**Comment by Mayousk** [comment]
Hello,
so i have recently been working on a problem with categorical , continuous and integer variables.Therefore, i used optimize_acqf_mixed which takes as an argument the bounds related to the variables.However,if i want to use one hot encoding meaning assigning a vector of one and zeros to each category , i won't be able to fully define the bounds for the categorical variables.So basically, the only way to use this function is to assign a numerical representation to the categorical variables which introduces an order ( that doesn't truly exist)  . Is there anyway around it? Also, there seems to be a problem in the link of the source code for the function.
thank you!
